spec/support/fakes/bin/heroku

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+
+require "pathname"
+require Pathname(__dir__) / ".." / ".." / "fake_heroku"
```

Comment:
`heroku` and `hub` fakes already existed before. I think GH is showing a green diff because I refactored them to use `Pathname` instead of the old `File` API.